modules/apps/blogs/blogs-editor-configuration/src/main/java/com/liferay/blogs/editor/configuration/internal/BlogsCoverImageCaptionEditorConfigContributor.java

```diff
@@ -32,6 +32,7 @@
 @Component(
 	property = {
 		"editor.config.key=coverImageCaptionEditor",
+		"editor.name=ballooneditor",
```

Comment:
If we are setting editor name here, we don't have to set it in JSP. It's probably better to leave the update in JSP, because it is consistent with setting in JSP when editor is not hardcoded. Normally it's not hardcoded, but set in `portal-ext.properties`.

Reply:
Nice! I removed `"editor.name=ballooneditor",` from OSGi properties and It works fine.